### PR TITLE
Made NavBar Font Consistent (#445)

### LINKED ISSUE #445: NavBar with Inconsistent Font size and Spacing
**Actual Behaviour**
NavBar with Inconsistent Font size and Spacing

**Expected Behaviour**
Should be consistent 

**Steps to reproduce it**
https://gci18.fossasia.org/

**Screenshots of the issue**
screenshot from 2018-11-22 09-59-10

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by niteshkumarniranjan**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://niteshkumarniranjan.github.io/gci18.fossasia.org/)

- Fixes #445 

### Before:
screenshot from 2018-11-23 19-13-52

### After:
screenshot from 2018-11-23 19-22-20

- - - - - - - - - - - -

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -98,14 +97,18 @@
...
-                <a href="https://twitter.com/fossasia" target="_blank" rel="noopener"><i class="fa fa-twitter fa-lg fa-fw"
-                    aria-hidden="true"></i></a>
-                <a href="https://facebook.com/fossasia" target="_blank" rel="noopener"><i class="fa fa-facebook fa-lg fa-fw"
-                    aria-hidden="true"></i></a>
-                <a href="https://github.com/fossasia" target="_blank" rel="noopener"><i class="fa fa-github fa-lg fa-fw"
-                    aria-hidden="true"></i></a>
+                <a href="https://twitter.com/fossasia" target="_blank" rel="noopener"><i class="fab fa-twitter fa-lg fa-fw"
+                    aria-hidden="true"></i>
+                </a>
+                <a href="https://facebook.com/fossasia" target="_blank" rel="noopener"><i class="fab fa-facebook fa-lg fa-fw"
```
Comment (abishekvashok) [inline _includes/header.html]: What are these changes about

**Comment by niteshkumarniranjan** [comment]
@abishekvashok Actually I was also working on another issue and I pushed that also by mistake. but I have reverted it back.

File: css/custom.css [hunk 2]
```diff
@@ -189,10 +187,10 @@ p.person-name {
   font-family: Helvetica, Arial, sans-serif;
   color: black;
 }
- .social-media-title:hover {
```
Comment (Masquerade0097) [inline css/custom.css]: Not needed.

File: css/custom.css [hunk 3]
```diff
@@ -262,104 +260,115 @@ p.person-name {
   opacity: 1;
   visibility: visible;
 }
-#img_project{
+#img_project {
```
Comment (Masquerade0097) [inline css/custom.css]: NN

File: css/custom.css [hunk 4]
```diff
@@ -262,104 +260,115 @@ p.person-name {
...
 }
-.team_member_name{
+.team_member_name {
   font-size: 18px;
   margin-top: 5px;
   font-weight: bold;
   padding-bottom: 20px;
 }
-.team_member_designation{
+.team_member_designation {
```
Comment (Masquerade0097) [inline css/custom.css]: You are not supposed to Re-indent entire file when you commit. Only indent changes that you've done.

**Comment by niteshkumarniranjan** [comment]
Closing in favor of #454

**Pull request closed without merging**